balancer/roundrobin/roundrobin.go

```diff
@@ -59,44 +59,24 @@ func (bb builder) Name() string {
 
 func (bb builder) Build(cc balancer.ClientConn, opts balancer.BuildOptions) balancer.Balancer {
 	bal := &rrBalancer{
-		cc:    cc,
-		child: endpointsharding.NewBalancer(cc, opts),
+		cc:       cc,
+		Balancer: endpointsharding.NewBalancer(cc, opts),
 	}
 	bal.logger = internalgrpclog.NewPrefixLogger(logger, fmt.Sprintf("[%p] ", bal))
 	bal.logger.Infof("Created")
 	return bal
 }
 
 type rrBalancer struct {
+	balancer.Balancer
 	cc     balancer.ClientConn
-	child  balancer.Balancer
 	logger *internalgrpclog.PrefixLogger
 }
 
-func (b *rrBalancer) Close() {
-	b.child.Close()
-}
-
-func (b *rrBalancer) ExitIdle() {
-	// Should always be ok, as child is endpoint sharding.
-	if ei, ok := b.child.(balancer.ExitIdler); ok {
-		ei.ExitIdle()
-	}
-}
```

Comment:
This is still necessary, unfortunately, as only `balancer.Balancer` functions are transferred when embedding.

Reply:
Reverted.